examples/deadline/All-In-AWS-Infrastructure-SEP/ts/README.md

```diff
@@ -17,12 +17,50 @@ These instructions assume that your working directory is `examples/deadline/All-
     ```
     yarn install
     ```
-3. Modify the `deadline_ver` field in the `config` block of `package.json` as desired (Deadline 10.1.9 and up are supported), then stage the Docker recipes for `RenderQueue`:
+3. Modify the `deadline_ver` field in the `config` block of `package.json` as desired (Deadline 10.1.12 and up are supported), then stage the Docker recipes for `RenderQueue`:
```

Comment:
I will reply on this comment and will update the code when the decision on how to bump the minimal required Deadline version is made. The meeting is planned for February, 3.

Reply:
Added a strict version check when using `ConfigureSpotEventPlugin` construct.